Blizzard.IO.RabbitMQ.Tests/NetqRabbitPublisherTests.cs

```diff
@@ -0,0 +1,112 @@
+﻿using Blizzard.IO.RabbitMQ.Entities;
+using EasyNetQ;
+using EasyNetQ.Topology;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using ISerializer = Blizzard.IO.Core.ISerializer<string>;
+
+namespace Blizzard.IO.RabbitMQ.Tests
+{
+    [TestFixture]
+    class NetqRabbitPublisherTests
```

Comment:
You are not testing the scenario where the `isAbstarct` field is `true`

Reply:
F